tm2/pkg/bft/rpc/lib/server/handlers.go

```diff
@@ -145,6 +145,9 @@ func makeJSONRPCHandler(funcMap map[string]*RPCFunc, logger *slog.Logger) http.H
 			requests  types.RPCRequests
 			responses types.RPCResponses
 		)
+
+		isRPCRequests := true
```

Comment:
I don't understand the purpose of this variable. If the point is to know if there is only one request (so only one response) why don't you use a `len(requests)` instead?
Or event better, a `len(responses)` in your case (since 1 request == 1 response if I read the for loop bellow correctly).

If there's another purpose, please change the variable name and add a comment to explain it.

Reply:
Thanks for the review! ^^

In this PR, I've updated the logic for handling batch requests. Previously, if a batch request contained only a single item, the endpoint would return just a single response object instead of an array containing that response. This was inconsistent with the expected behavior where even a single-item batch should yield an array of responses.
With my changes, regardless of the number of items in the batch request, the API now consistently returns an array.